### PR TITLE
kcat: add page

### PR DESCRIPTION
**Pull request by ImShakthi**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/kcat.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+> Apache Kafka producer and consumer tool.
+> More information: <https://github.com/edenhill/kcat>.
+
+- Consume messages starting with the newest offset:
+
+`kcat -C -t {{topic}} -b {{brokers}}`
+
+- Consume messages starting with the oldest offset and exit after the last message is received:
+
+`kcat -C -t {{topic}} -b {{brokers}} -o beginning -e`
```
Comment (marchersimon) [inline pages/common/kcat.md:12]:
```suggestion
`kcat -C -t {{topic}} -b {{brokers}} -o {{beginning}} -e`
```

File: pages/common/kcat.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Consume messages as a Kafka consumer group:
+
+`kcat -G {{group_id}} -t {{topic}} -b {{brokers}}`
+
+- Publish message by reading from stdin:
```
Comment (marchersimon) [inline pages/common/kcat.md:18]:
```suggestion
- Publish a message, reading it from stdin:
```

File: pages/common/kcat.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+` echo {{message}} | kcat -P -t {{topic}} -b {{brokers}}`
+
+- Publish messages by reading from a file:
```
Comment (marchersimon) [inline pages/common/kcat.md:22]:
```suggestion
- Publish messages, reading them from a file:
```

File: pages/common/kcat.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- List metadata for all topics and brokers:
+
+`kcat -L -b {{brokers}}`
+
+- List metadata for a specific topic:
+
+`kcat -L -t {{topic}} -b {{brokers}}`
+
+- Get offset for a topic/partition for a specific point in time:
```
Comment (marchersimon) [inline pages/common/kcat.md:34]:
```suggestion
- Get the offset for a topic or partition for a specific point in time:
```

**Review by marchersimon — COMMENTED** [review]
Just a few small suggestions

**Comment by navarroaxel** [comment]
@ImShakthi Could you please apply the last change?

**Comment by navarroaxel** [comment]
@ImShakthi could you help us to merge this PR?

**Pull request closed without merging**